db/migrate/20230326064945_create_chhands.rb

```diff
@@ -0,0 +1,13 @@
+class CreateChhands < ActiveRecord::Migration[7.0]
+  def change
+    create_table :chhands do |t|
+      t.references :chapter, :foreign_key => true, :null => false
```

Comment:
Thanks for contributing! 

This looks great! 

We actually don't need a `t.string chhand_type` and `t.string en_chhand_type` attribute on here, because now that you add the `chhand_type` reference, 

we will eventually be able to do something like:

`@chhand.chhand_type.chhand_name #> "ਦੋਹਰਾ"`

If you do want to store the English one, we can add `en_chhand_name` on the `chhand_type` table :)